suse/macros

```diff
@@ -230,6 +230,11 @@
 %cflags_profile_generate -fprofile-update=atomic -fprofile-generate
 %cflags_profile_feedback -fprofile-use
 
+# Reproducible builds
+%build_mtime_policy clamp_to_buildtime
+%_buildtime %{getenv:SOURCE_DATE_EPOCH_MTIME}
+%_buildhost reproducible
```

Comment:
These two are inappropriate to force for all build environments everywhere. This should be split out into another macro file and subpackaged out.

Reply:
Done